index.html

```diff
@@ -413,6 +395,46 @@ <h3 class="question__title">JSX</h3>
 
       <div id="q13" class="question">
         <a href="#q13" class="question__num">13.</a>
+        <div class="question__body">
+          <h3 class="question__title">useEffect hook</h3>
+          <div class="question__text">
+            <p>Jaké je správné použití useEffect hooku?</p>
```

Comment:
Upravil bych na:

> Jaké je správné použití `useEffect` hooku pro akci při prvním zobrazení komponenty?

A možná bych ještě doplnil poznámku:

> Uvažujeme jednoduché užití `useEffect` hooku, kdy se nevolá API a není tedy nutné použít `async/await`.

Protože to `async/await` s `useEffectem` jsme dost zdůrazňovali a dost je to potrápilo, tak ať vědí, že se tady ptáme na něco jednoduššího.